javascript/javascript1/week1/house.js

```diff
@@ -0,0 +1,34 @@
+// A house price estimator
+
+  // for Peter
+  let PeterVolumeInMeters=10*10*8;
```

Comment:
Variable names should start with lowercase letters (e.g., peterVolumeInMeters instead of PeterVolumeInMeters) same with lines 5, 7, 20, 21, 22 and 23